src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/TupleTests.cs

```diff
@@ -261,8 +261,8 @@ .maxstack  6
   IL_002f:  ldstr      ""y""
   IL_0034:  call       ""(int A, int B) Microsoft.VisualStudio.Debugger.Clr.IntrinsicMethods.GetVariableAddress<(int A, int B)>(string)""
   IL_0039:  ldloc.0
-  IL_003a:  stind.ref
-  IL_003b:  ret
+  IL_003a:  stobj      ""System.ValueTuple<int, int>""
```

Comment:
Tuple type was not a struct here

---
In reply to: [81644123](https://github.com/dotnet/roslyn/pull/14161#discussion_r81644123) [](ancestors = 81644123)